content/en/releases/version-skew-policy.md

```diff
@@ -45,12 +45,13 @@ Example:
 
 ### kubelet
 
-`kubelet` must not be newer than `kube-apiserver`, and may be up to two minor versions older.
+* `kubelet` must not be newer than `kube-apiserver`.
+* `kubelet` may be up to three minor versions older than `kube-apiserver` (`kubelet` < 1.25 may only be up to two minor versions older than `kube-apiserver`).
```

Comment:
Can we reword this so that it won't need to be changed in 1.29? i.e. Rather saying `kubelet < 1.25`, give it as an example or use templating like below.

Reply:
it won't need to be reworded... the < 1.25 condition is the fixed point at which skew support expanded

Reply:
Ah, misread that. You're right. Tx for clarifying. Ship it!